src/components/anotherlist.js

```diff
@@ -1,198 +1,67 @@
-                                 <div className="details">
-                                     <img src='./images/tomtoes.jpg' alt="#" style={{width: '150px', height: '115px'}} />
-                                 </div>
-                                                                   &nbsp;&nbsp;&nbsp;
-                                 <div className="details">
-                                      <p><b>Tomatoes</b><br />Location: Lagos<br />Name: My Store<br />#4,599 per basket</p>
-                                 </div>
-                           </Col>
-                           <Col md={3}>
-                                <div className="details">
-                                     <img src='./images/beans.jpg'alt="#"  style={{width: '155px', height: '115px'}} />
-                                </div>
-                                                                       &nbsp;&nbsp;&nbsp;
-                                <div className="details">
-                                    <p><b>Maize</b><br />Location: Gurara, Niger State<br />Name: My Store<br />#11,500</p>
-                                </div>
-                           </Col>
-                           <Col md={3}>
-                                <div className="details">
-                                     <img src='./images/ogbono.jpg' alt="#" style={{width: '150px', height: '115px'}} />
-                                </div>
-                                                                      &nbsp;&nbsp;&nbsp;
-                                <div className="details">
-                                     <p><b>Docanut(Ogbono)</b><br />Location: Idumota, Lagos State<br />Name: My Store<br />#2,500</p>
-                                </div>
-                          </Col>
-                          <Col md={3}>
-                              <div className="details">
-                                    <img src='./images/ripeBanana.jpg' alt="banana" style={{width: '150px', height: '100px'}} />
-                              </div>
-                                                                     &nbsp;&nbsp;&nbsp;
-                              <div className="details">
-                                  <p><b>Ripe Bananas</b><br />Location: Ore, Ondo State<br />Name: My Store<br />#1,250</p>
-                              </div>
-                          </Col>
-                   </Row>
-                         <div className="link-to-more">
-                               <Link to='/more-products/'>
-                                    More
-                               </Link>
-                         </div>
-              </Jumbotron>
-        </Container>
+        <div>
+           <Jumbotron className="show-products-available" style={{backgroundColor: 'ash'}}>
+             <Row>
+                <Col md={3}>
+                    <div>
+                      <div  style={{backgroundColor: 'white'}}>
```

Comment:
no in-line styles, give this div a className this define the styles in a `css` file

Reply:
Ok